dev/AccessControl/Microsoft.Windows.Security.AccessControl.cpp

```diff
@@ -0,0 +1,50 @@
+﻿#include <pch.h>
+#include "Microsoft.Windows.Security.AccessControl.h"
+#include "Microsoft.Windows.Security.AccessControl.SecurityDescriptorHelpers.g.cpp"
+#include "Security.AccessControl.h"
+#include <sddl.h>
+
+namespace winrt::Microsoft::Windows::Security::AccessControl::implementation
+{
+    wil::unique_hlocal_security_descriptor GetSecurityDescriptor(array_view<winrt::Microsoft::Windows::Security::AccessControl::AppContainerNameAndAccess const> accessRequests, hstring const& principalStringSid, uint32_t principalAccessMask, DWORD* sdLength)
+    {
+        wil::unique_any_psid sid;
+        if (!principalStringSid.empty())
+        {
+            winrt::check_hresult(::ConvertStringSidToSidW(principalStringSid.data(), &sid));
```

Comment:
THROW_IF_WIN32_BOOL_FALSE()

ConvertStringSidToSidW returns BOOL/LastError. Not HRESULT

https://docs.microsoft.com/en-us/windows/win32/api/sddl/nf-sddl-convertstringsidtosidw